src/pages/api/blogs/index.js

```diff
@@ -21,6 +25,14 @@ const handler = (req, res) => {
       .catch((error) =>
         res.status(400).json({ success: false, message: error.message })
       );
+  } else if (req.method === "PUT") {
+    if (req.body.action === "REORDER_BLOGS") {
```

Comment:
Added this logic so that if we were to perform some other bulk update for all of the blogs, we can simply send a PUT request to `/api/blogs` and pass `action` in the request body. Based on the value for `action` it will carry out the required functionality.